VMEncryption/main/patch/centosPatching.py

```diff
@@ -73,40 +73,17 @@ def __init__(self, logger, distro_info):
             self.resize2fs_path = '/sbin/resize2fs'
             self.umount_path = '/usr/bin/umount'
 
-    def install_extras(self):
-        epel_packages_installed = False
-        attempt = 0
-
-        while not epel_packages_installed:
-            attempt += 1
-            self.logger.log("Attempt #{0} to locate EPEL packages".format(attempt))
-            if self.distro_info[1].startswith("6."):
-                if self.command_executor.Execute("rpm -q ntfs-3g python-pip"):
-                    epel_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm"
-
-                    if self.command_executor.Execute("rpm -q epel-release"):
-                        self.command_executor.Execute(epel_cmd)
-
-                    self.command_executor.Execute("yum install -y ntfs-3g python-pip")
-
-                    if not self.command_executor.Execute("rpm -q ntfs-3g python-pip"):
-                        epel_packages_installed = True
-                else:
-                    epel_packages_installed = True
-            else:
-                if self.command_executor.Execute("rpm -q ntfs-3g python2-pip"):
-                    epel_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
-
-                    if self.command_executor.Execute("rpm -q epel-release"):
-                        self.command_executor.Execute(epel_cmd)
+    def install_cryptsetup(self):
+        packages = ['cryptsetup',
+                    'cryptsetup-reencrypt']
```

Comment:
>cryptsetup-reencrypt [](start = 21, length = 20)

do we use cryptsetup-reencrypt for anything?

Reply:
This package is used by _luks_reencrypt in EncryptBlockDeviceState.py